content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -108,32 +110,40 @@ cluster for mesh expansion, run the following commands on a machine with cluster
           -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
     {{< /text >}}
 
-### Setting up the machines
+### Setting up VM machines
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Install the Debian package with the Envoy sidecar:
+<!-- TODO: find a better way to reference release in the URL instead of manually sync... -->
+
+1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM. For example
 
     {{< text bash >}}
-    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ dpkg -i istio-sidecar.deb
+    $ export GCE_NAME="your-gce-instance"
+    $ gcloud compute scp {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
     {{< /text >}}
 
-1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM.
+1.  Install the Debian package with the Envoy sidecar.
+
+    {{< text bash >}}
+    $ gcloud compute ssh "${GCE_NAME}"
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.1.0-snapshot.6/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
+    {{< /text >}}
 
 1.  Add the IP address of the Istio gateway (which we found in the [previous section](#preparing-the-kubernetes-cluster-for-expansion))
-    to `/etc/hosts` or to
-    the DNS server. In our example we'll use `/etc/hosts` as it is the easiest way to get things working. The following is
-    an example of updating an `/etc/hosts` file with the Istio gateway address:
+to `/etc/hosts` or to
+the DNS server. In our example we'll use `/etc/hosts` as it is the easiest way to get things working. The following is
+an example of updating an `/etc/hosts` file with the Istio gateway address:
 
     {{< text bash >}}
-    $ echo "35.232.112.158 istio-citadel istio-pilot istio-pilot.istio-system" >> /etc/hosts
+    $ sudo echo "35.232.112.158 istio-citadel istio-pilot istio-pilot.istio-system" >> /etc/hosts
```

Comment:
This command will not work. Use `tee`.

Reply:
I used this and it works. Not sure what you mean not working.

Well it does give the error about permission not enough, and works if we change to `sudo su`. But that's not related to 'tee|echo".

Reply:
The stdout redirection, ">>", takes place in the non-root shell, the `sudo` command never even sees that redirection command. The above command just executes `echo` as root, and then the non-root shell attempts to append to `/etc/hosts`, which is going to fail unless you're running sudo as root. Changing the command to `sudo su` does not fix this problem for the same reasons.

One way to solve this to do: `echo "1.2.3.4 blah" | sudo tee -a /etc/hosts`.

Reply:
Thanks for explanation! Changed and verified on a VM machine.